### ISSUE TITLE
Wrong place for exclamation mark

### ISSUE THREAD

**Issue by piotrgorecki** [APIs-guru/graphql-voyager#49]
In my schema I have `groups: [CustomerGroup!]`, but Voyager display it as "groups: [CustomerGroup **]!**".
The opposite case, when I set in the schema `groups: [CustomerGroup]!`, the Voyager shows it as "groups: [CustomerGroup **!]**"

```
"apollo-server-express": "^1.3.0",
"babel-cli": "^6.26.0",
"babel-core": "^6.26.0",
"babel-plugin-module-resolver": "^3.0.0",
"babel-preset-es2015": "^6.24.1",
"babel-preset-stage-2": "^6.24.1",
"express": "^4.16.2",
"graphql": "^0.11.7",
"graphql-tools": "^2.13.0",
"graphql-voyager": "^1.0.0-rc.10",
```

```js
app.use("/voyager", middleware({ endpointUrl: "/graphql" }));
```

**Comment by IvanGoncharov** [comment]
@piotrgorecki Good catch  🐞 🔍
Fixed in `1.0.0-rc.11` and demo.

**Comment by piotrgorecki** [comment]
Thanks @IvanGoncharov! Now it works correctly.
